modules/capi-yaml-machine-set-rhosp.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster_api_provider_configurations/cluster-api-config-options-rhosp.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="capi-yaml-machine-set-rhosp_{context}"]
+= Sample YAML for a Cluster API compute machine set resource on {rh-openstack}
+
+The compute machine set resource defines additional properties of the machines that it creates.
+The compute machine set also references the infrastructure resource and machine template when creating machines.
+
+[source,yaml]
+----
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineSet
+metadata:
+  name: <machine_set_name> # <1>
+  namespace: openshift-cluster-api
+spec:
+  clusterName: <cluster_name> # <2>
+  replicas: 1
+  selector:
+    matchLabels:
+      test: example
+  template:
+    metadata:
+      labels:
+        test: example
+    spec:
+      bootstrap:
+         dataSecretName: worker-user-data # <3>
+      clusterName: <cluster_name>
+      infrastructureRef:
+        apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+        kind: OpenStackMachineTemplate # <4>
+        name: <template_name> # <5>
+      failureDomain: # <6>
+----
+<1> Specify a name for the compute machine set.
+<2> Specify the cluster ID as the name of the cluster.
+<3> For the Cluster API Technology Preview, the Operator can use the worker user data secret from the `openshift-machine-api` namespace.
+<4> Specify the machine template kind.
+This value must match the value for your platform.
+<5> Specify the machine template name.
+<6> Specify the failure domain configuration details. #Need more detail here#
```

Comment:
This is the name of a Nova Availability Zone which all machines in the machineset would be created in.

If the user does not want to confine machine to a specific availability zone they should omit this value. I expect this value to be omitted by most users.

Reply:
ok cool - I saw in the repo it said "string", so do we just need something like `<nova_availability_zone>` for the value here?